tests/test_converter.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+import os
+import sys
+import shutil
+import json
+from converter import Converter
+
+
+class TestConverter(unittest.TestCase):
+    def setUp(self):
+        self.content = '<html><body></body></html>'
+        self.page_path = '/tmp/test/'
+        if not os.path.exists('/tmp/test/depth'):
+            os.makedirs('/tmp/test/depth')
+        self.hname1 = ""
+        self.hname2 = ""
+        with open(os.path.join(self.page_path, 'index.html'), 'w') as f:
+            f.write(self.content)
+        with open(os.path.join(self.page_path, 'depth/page.html'), 'w') as f:
+            f.write(self.content)
+        self.cnv = Converter()
+
+    def test_converter(self):
+        self.cnv.convert(self.page_path)
+        with open(os.path.join(self.page_path, 'meta.json')) as f:
+            s = json.loads(f.read())
```

Comment:
please use `json.load(f)` here :)

Json has two function: `load` and `loads`:
`load` can directly read from text stream
`loads` read json from a string